services/cert-manager/1.11.0/defaults/cm.yaml

```
@@ -4,5 +4,7 @@ metadata:
   name: cert-manager-1.11.0-d2iq-defaults
   namespace: ${releaseNamespace}
 data:
-  values.yaml: ""
+  values.yaml: |-
+    global:
+      priorityClassName: dkp-high-priority
```

we decided to use `system-cluster-critical` for cert-manager (https://d2iq.atlassian.net/browse/D2IQ-96866)
```suggestion
      priorityClassName: system-cluster-critical
```